compiler-core/src/ast.rs

```diff
@@ -605,11 +613,13 @@ pub type UntypedTypeAlias = TypeAlias<()>;
 pub struct TypeAlias<T> {
     pub location: SrcSpan,
     pub alias: EcoString,
+    pub alias_location: SrcSpan,
```

Comment:
I was a bit confused by this as it's `alias.alias_location`. Is it the name, or the whole alias definition?

Reply:
It's just the name! I wanted it to be consistent with the `alias` field, so I guess it suffers from the same problem haha. Maybe we could rename both to `name`?

Reply:
This field no longer exists and was merged with `alias`, so I'll resolve the thread. Feel free to unresolve if further action should be taken.